hicexplorer/trackPlot.py

```diff
@@ -264,31 +267,47 @@ def parse_tracks(self, tracks_file):
                     track_dict['title'] = ''
                 if warn:
                     sys.stderr.write(warn)
-
-        self.track_list = track_list
+            updated_track_list.append(track_dict)
+        self.track_list = updated_track_list
         if self.vlines_properties:
             self.vlines_intval_tree, __, __ = file_to_intervaltree(self.vlines_properties['file'])
 
     @staticmethod
-    def check_file_exists(track_dict):
+    def check_file_exists(track_dict, tracks_path):
         """
-        Checks if a file or list of files exists
+        Checks if a file or list of files exists. If the file does not exists
+        tries to check if the file may be relative to the track_file path, in such case
+        the path is updated.
         :param track_dict: dictionary of track values. Should contain
                             a 'file' key containing the path of the file
                             or files to be checked separated by space
                             For example: file1 file2 file3
+        :param tracks_path: path of the tracks file
         :return: None
         """
-        file_names = [x for x in track_dict['file'].split(" ") if x != '']
-        for file_name in file_names:
-            try:
-                open(file_name, 'r').close()
-            except IOError:
-                sys.stderr.write("\n*ERROR*\nFile in section [{}] "
-                                 "not found:\n{}\n\n".format(track_dict['section_name'],
-                                                             file_name))
-                sys.exit(1)
-
+        for file_type in ['boundaries_file', 'file']:
+            if file_type in track_dict:
+                file_names = [x for x in track_dict[file_type].split(" ") if x != '']
+                full_path_file_names = []
+                for file_name in file_names:
+                    try:
+                        open(file_name, 'r').close()
```

Comment:
https://docs.python.org/2/library/os.path.html#os.path.exists does not work here?